src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/avoidnoargumentsuperconstructorcall/Example1.java

```diff
@@ -0,0 +1,32 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="AvoidNoArgumentSuperConstructorCall"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.avoidnoargumentsuperconstructorcall;
+
+class SomeOtherClass {
+  public SomeOtherClass() {}
+
+  public SomeOtherClass(int arg) {}
+}
+
+
```

Comment:
There is only one example file so no need to introduce unnecessary blank lines. Please remove